apps/src/gamelab/AnimationPicker/AnimationPreview.jsx

```diff
@@ -56,6 +58,16 @@ const AnimationPreview = React.createClass({
   },
 
   advanceFrame: function () {
+    // If the animation shouldn't loop, include a 2.5 second timeout after playing.
+    if (this.props.animationProps.looping === false && (this.state.currentFrame % this.props.animationProps.frameCount) === 0) {
```

Comment:
Why not just `!this.props.animationProps.looping` here?

Also, this is a really long line.  We don't intend to mutate any of these variables in this method, maybe we should use local `const`s to clean this up, e.g.

```javascript
const {currentFrame} = this.state;
const {frameCount, looping} = this.props.animationProps;
if (!looping && currentFrame % frameCount === 0) {
  // ...could replace long property chains in here too
}
```